code/modules/awaymissions/mission_code/UO71-terrorspiders.dm

```diff
@@ -191,26 +191,28 @@
 	origin_tech = null
 	selfcharge = 1
 	can_charge = 0
-	var/inawaymission = 1
+	var/inawaymission = TRUE
 
-/obj/item/gun/energy/laser/awaymission_aeg/process()
+/obj/item/gun/energy/laser/awaymission_aeg/Initialize(mapload)
+	. = ..()
+	RegisterSignal(src, list(COMSIG_MOVABLE_Z_CHANGED), .proc/updateZ)
```

Comment:
I'm curious as to why the signal is in a list here. It probably works fine but, just wondering.

Reply:
Other examples in the code work like this, and I was concerned if something would break if I didn't do it like this.

Reply:
If it's just a single signal, you can just put the signal in.